apps/tangle-dapp/app/bridge/hooks/useBridgeTransfer.ts

```diff
@@ -42,9 +46,133 @@ export default function useBridgeTransfer({
   const { sourceTypedChainId, destinationTypedChainId } = useTypedChainId();
   const { sourceAmountInDecimals, destinationAmountInDecimals } =
     useAmountInDecimals();
+  const {
+    addTxToQueue,
+    addSygmaTxId,
+    updateTxState,
+    updateTxDestinationTxState,
+    addTxDestinationTxExplorerUrl,
+    addTxExplorerUrl,
+  } = useBridgeTxQueue();
+
+  const tangleTestnetMailboxAddress =
+    '0x0FDc2400B5a50637880dbEfB25d631c957620De8';
+  const holeskyMailboxAddress = '0x57529d3663bb44e8ab3335743dd42d2e1E3b46BA';
```

Comment:
Please add these addresses to the config or constants file for easier future access and updates.